stack.yaml.lock

```diff
@@ -3,7 +3,35 @@
 # For more information, please see the documentation at:
 #   https://docs.haskellstack.org/en/stable/lock_files
 
-packages: []
+packages:
+- completed:
+    hackage: basement-0.0.12@sha256:cbb9187d3575b78ee1165b7a49a0e8570ee563820330a944a67cd09e1cfe23bf,5516
+    pantry-tree:
+      size: 5859
+      sha256: 6462393c9058b93baea2c8ffb4a7e1d68c2aee14746ef0081dbb485a2faef71b
+  original:
+    hackage: basement-0.0.12
+- completed:
```

Comment:
I imagine you'll probably want to remove the changes in this file, since they look like they are package versions for ` stack-ghc-9.yaml`.